### DISCUSSION TITLE
求助！

### DISCUSSION THREAD

**Discussion by kunlinm** [share121/inter-knot#315] [其他]
挺急的外面有三大巡猎令使，还有个发魔芋爽的疯女人，求救求救我该如何越狱呢？
对了幽囚狱里面还想还有个戴礼帽的开拓者，还有个玩
悠悠球看起来像龙的男人，还有个黑不拉几好像能隐身的家伙
求救求救，有哪位绳匠能给指一条明路，呼雷事后必有**重谢！！！！！！！**
![image](https://github.com/user-attachments/assets/47b5c11f-2fbe-488a-a25b-0e91c9ff4d46)

**Comment by kianaS0** [comment]
这个委托我接了！
我找了一个很强的帮手给你，你出了幽囚狱走到神策府附近，那里有个白色头发的狐人女士（血统是伪装的），她很强，可以帮你脱离险境。

**Comment by dangerousNH4NO3** [comment]
高喊“左青龙右白虎，少吃碳水化合物”可以召唤一位身着仙舟服饰的粉毛少女剑士帮你逃离哦

**Comment by lappslover** [comment]
没救了等死吧，我一会叫十王司给你收尸

**Comment by xgsxjhlb** [comment]
药师万岁！
